test/test-spec/split/tags/openapi_spec.md

```diff
@@ -0,0 +1 @@
+OpenAPI spec for..
```

Comment:
Here's the original content (not sure where I found it...):

> The goal of The OpenAPI Specification is to define a standard, language-agnostic interface to REST APIs which allows both humans and computers to discover and understand the capabilities of the service without access to source code, documentation, or through network traffic inspection. When properly defined via OpenAPI, a consumer can understand and interact with the remote service with a minimal amount of implementation logic. Similar to what interfaces have done for lower-level programming, OpenAPI removes the guesswork in calling the service.

Reply:
Done